coda/coda_mdstore/presentation.py

```diff
@@ -79,6 +81,33 @@ def getFileHandle(codaId, codaPath):
     )
 
 
+def file_chunk_generator(url):
+    """
+    Download a file and stream it
+    """
+    r = requests.get(url, stream=True)
+    if r.status_code != 200:
+        return
+    for chunk in r.iter_content(1024):
+        yield chunk
+
+
+def zip_file_streamer(urls, identifier):
+    """
+    Stream downloadable zipped file using zipstream
```

Comment:
We can remove "downloadable", I think in this context it's redundant with "Stream".